lib/WP_Auth0_LoginManager.php

```diff
@@ -355,15 +359,16 @@ public function implicit_login() {
 	 * Attempts to log the user in and create a new user, if possible/needed.
 	 *
 	 * @param object $userinfo - Auth0 profile of the user.
-	 * @param string $id_token - user's ID token returned from Auth0.
-	 * @param string $access_token - user's access token returned from Auth0; not provided when using implicit_login().
+	 * @param null|string $id_token - user's ID token returned from Auth0.
+	 * @param null|string $access_token - user's access token returned from Auth0; not provided when using implicit.
+	 * @param null|string $refresh_token - user's refresh token returned from Auth0; not provided when using implicit.
 	 *
 	 * @return bool
 	 *
 	 * @throws WP_Auth0_LoginFlowValidationException - OAuth login flow errors.
 	 * @throws WP_Auth0_BeforeLoginException - Errors encountered during the auth0_before_login action.
 	 */
-	public function login_user( $userinfo, $id_token, $access_token ) {
+	public function login_user( $userinfo, $id_token = null, $access_token = null, $refresh_token = null ) {
```

Comment:
Since you're not using the values but passing them to the `do_login` function, shouldn't these defaults go in that function?

Reply:
This is the public method, `do_login` is private. We don't need to call `login_user` with values here so I don't want to require them.